app/models/issue_template_setting.rb

```diff
@@ -24,13 +24,13 @@ def enable_help?
     if self.enabled == true && !self.help_message.blank?
       return true
     end
-    return false
+    false
   end
 
   def enabled_inherit_templates?
-    if self.inherit_templates == true
+    if self.inherit_templates

```

Comment:
__[rubocop]__
* Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. [:ref](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)
* Redundant `self` detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-self-unless-required)